impeller/display_list/aiks_dl_blend_unittests.cc

```diff
@@ -566,5 +570,122 @@ TEST_P(AiksTest, FramebufferAdvancedBlendCoverage) {
   ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
 }
 
+TEST_P(AiksTest, ColorWheel) {
+  // Compare with https://fiddle.skia.org/c/@BlendModes
+
+  BlendModeSelection blend_modes = GetBlendModeSelection();
+
+  auto draw_color_wheel = [](DisplayListBuilder& builder) {
```

Comment:
nit: Since we are using `auto` we should have explicit return types in the lamda declaration, especially since the return is so far away.

Reply:
Should that be done by making the `auto` explicit or by the lambda return specifier `->`?

The `->` format doesn't stand out very well for me, but it is more explicit than just declaring a type for the variable.

Reply:
Oh wait, the `auto` here is not the return value of the lambda function, it's the type of the lambda itself? In that case `->` is much less disruptive.

Reply:
Yea, I was thinking `->`

Reply:
its a void function :)